src/main/java/no/nav/veilarbperson/client/kontoregister/HentKontoRequestDTO.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.veilarbperson.client.kontoregister;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class HentKontoRequestDTO {
+    private String kontohaver;
+    public String getKontohaver() {
```

Comment:
I think getter and setter function is not necessary, since its included with @Data (ref: https://projectlombok.org/features/Data)

Reply:
Removed.